ui/tests/integration/components/replication-dashboard-test.js

```diff
@@ -29,15 +31,9 @@ module('Integration | Enterprise | Component | replication-dashboard', function(
     @clusterMode={{clusterMode}}
     @isSecondary={{isSecondary}}
     />`);
-
     assert.dom('[data-test-replication-dashboard]').exists();
     assert.dom('[data-test-table-rows').exists();
-    assert
-      .dom('[data-test-primary-cluster-address]')
-      .includesText(
-        REPLICATION_DETAILS.primaryClusterAddr,
-        `shows the correct primary cluster address value`
-      );
+    assert.dom('[data-test-selectable-card-container="secondary"]').exists();
```

Comment:
if this is testing that the right content is being rendered when the dashboard is for a secondary, we should also probably add a test case for when the dashboard is a primary. 

if it's a primary, for instance, it should render the `KnownSecondariesCard`: https://github.com/hashicorp/vault/blob/ade82196f1739b76795dd96136f9aee5e550eae4/ui/lib/core/addon/templates/components/replication-dashboard.hbs#L18